import os

os.path.join(1,2) #exception. where?